### DISCUSSION TITLE
API yields wierd output

### DISCUSSION THREAD

**Discussion by kyung01** [Bing-su/adetailer#96] [General]
When I use your extension through API it yields a weird result

This is the default
![image](https://github.com/Bing-su/adetailer/assets/6539484/eee8d497-644b-4e0d-a680-d8ef1fbbbd95)

This is when I do it on webUI automatic 1111
![image](https://github.com/Bing-su/adetailer/assets/6539484/6020367a-d8b8-4c9c-892b-eb3d74719bd4)

This is when I do it on API
![image](https://github.com/Bing-su/adetailer/assets/6539484/7c8a3141-7a1e-4a52-84cb-2bdbc09449bb)
![image](https://github.com/Bing-su/adetailer/assets/6539484/fb3d8f6d-7493-4d1a-9fb7-fefc81e8e130)
![image](https://github.com/Bing-su/adetailer/assets/6539484/4a4e6fd2-5f73-41e9-a84d-9fca9d39a86f)

ANOTHER EXAMPLE

DEFAULT:
![image](https://github.com/Bing-su/adetailer/assets/6539484/3f325dbf-c20c-4020-b45e-fa22f092de5f)
DETAILER:
![image](https://github.com/Bing-su/adetailer/assets/6539484/382325cf-b989-4660-8469-790c0d9eef98)
![image](https://github.com/Bing-su/adetailer/assets/6539484/337d710c-fa97-4c87-b638-d2e85e510ea1)

Your extension works fantastically on webUI but I don't know what I am doing wrong on API usage. It gets all distorted.
This is my payload.
```
payload = {
        "prompt": positive_sentence,
        "steps": 25,
        "negative_prompt": 'kid, children, low quality, poorly drawn. '+negative_sentence,
        "width" : width,
        "height" : height,
        "batch_size" : 1,
        "restore_faces" : False,
        "sampler_index" : "DPM++ SDE Karras",

        "alwayson_scripts": {
        "After Detailer": {
          "args": [
            #
            {
              "ad_model": "person_yolov8s-seg.pt",
            },
            {
              "ad_model": "face_yolov8n.pt"
            }
          ]
        }
      }
    }
```
    `

**Comment by Bing-su** [comment]
Is there a difference in the two images...?

**Reply by kyung01** [reply]
sorry I uploaded two same images

**Reply by kyung01** [reply]
@Bing-su fixed!

**Reply by Bing-su** [reply]
What extensions are you using?

**Reply by kyung01** [reply]
https://github.com/Bing-su/adetailer this extension

**Reply by Bing-su** [reply]
What other extensions are you using?

**Reply by kyung01** [reply]
Thats all. I use default Automatic1111

**Reply by Bing-su** [reply]
Looking at the example again now, I can see why. The border around the object is a common occurrence with this extension. It's likely that the API call just happened to output a little more of the bad picture. Adjusting the cfg scale, increasing the inpaint padding value, and using an extension like dynamic thresholding should help.

**Reply by kyung01** [reply]
Then why is it outputting a beautiful image when used on Automatic1111 UI? I don't use any other extensions on web UI as well. Plus the settings on webUI and settings on API call are all the same.
